README.md

```diff
@@ -97,10 +109,12 @@ Some READMEs also contain brief interactive elements such as quizzes or little
 in-browser coding challenges.
 
 Once you've completed a README, you should click the "I'm Done" button on the
-right. The "Next Lesson" button will light up, allowing you to proceed.
```

Comment:
My only feedback I see is that these feels like `The video augments the content; it does not replace it.` should have **bold** emphasis. I think this is something that should stick in the readers' mind.